docs/HailExpressionLanguage.md

```diff
@@ -56,6 +56,8 @@ Several Hail commands provide the ability to perform a broad array of computatio
      - max: `arr.max` -- valid only for numeric arrays, returns the minimum value
      - find: `arr.find(v => expr)` -- Returns the first non-missing element of `arr` for which `expr` is true.  If no element satisfies the predicate, `find` returns NA.
      - map: `arr.map(v => expr)` -- Returns a new array produced by applying `expr` to each element
+     - flatMap: `arr.flatMap(v => expr)` -- valid only for `expr` of type Array. Returns a new array by mapping each element of `arr` and concatenating the resulting arrays
```

Comment:
This comment applies to flatMap on arrays only, right? @danking and I thought that flatMap should be consistent on type from monad point of view.  But I agree that union of elements of array of sets is well-defined.  On the other hand, concatenation of elements of set of arrays is not.